### ISSUE TITLE
[Xamarin.Forms] SVG image rendering problem on Android

### ISSUE THREAD

**Issue by marlonconstante** [luberda-molinet/FFImageLoading#605]
I'm using the latest prerelease available **Xamarin.FFImageLoading.Svg.Forms 2.2.10-pre-390** together with **Xamarin.Forms 2.3.4.231**.

**Here is the XAML code used:**

```xml
<?xml version="1.0" encoding="UTF-8"?>
<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
    xmlns:ffimageloadingsvg="clr-namespace:FFImageLoading.Svg.Forms;assembly=FFImageLoading.Svg.Forms"
    x:Class="App.Views.MainPage">
    <ContentPage.Content>
        <ffimageloadingsvg:SvgCachedImage
                        HorizontalOptions="Center"
                        VerticalOptions="Center"
                        WidthRequest="200"
                        HeightRequest="200"
                        Source="http://thenewcode.com/assets/images/thumbnails/homer-simpson.svg"/>
    </ContentPage.Content>
</ContentPage>
```

Is there any known problem when rendering an SVG on Android?

**Follow print for comparison of the same screen on both platforms (iOS and Android)**
![screens](https://cloud.githubusercontent.com/assets/6370612/26061961/d0642704-395f-11e7-9b89-3d9d9121f07f.png)

**Comment by michelmoorlag** [comment]
I have the same issue and found that it is related to SkiaSharp 1.57.1! When you downgrade to Xamarin.FFImageLoading.Svg.Forms 2.2.10-pre-375 and then downgrade SkiaSharp to 1.56.1 it will work fine.

**Comment by kyurkchyan** [comment]
Having the same problem

**Comment by kyurkchyan** [comment]
Tried @michelmoorlag suggestion, but this brings back the issue [#522](https://github.com/luberda-molinet/FFImageLoading/issues/522). This is pretty serious, can't use the latest packages, again will need to switch to my private fork. Would be cool to see this fixed ASAP.

**Comment by fmeneuhe** [comment]
The same here.

**Comment by michelmoorlag** [comment]
I tried with 2.2.10-pre-391 with skiasharp 1.58.0 but the issue is not resolved (yet). Svg colors was messed up for android in SkiaSharp 1.57.1. I added a new issue there: 
https://github.com/mono/SkiaSharp/issues/300

**Comment by michelmoorlag** [comment]
It should be fixed in SkiaSharp 1.58.1. See https://github.com/mono/SkiaSharp/issues/284

**Comment by phamthanhtong** [comment]
@michelmoorlag : Thank you!
I'm waiting for package SkiaSharp 1.58.1

**Comment by daniel-luberda** [comment]
It should be fixed as 1.59.1 is current stable
